package.json

```diff
@@ -57,7 +57,7 @@
     "docs": "npm-run-all docs-build docs-lint",
     "docs-build": "hugo --cleanDestinationDir",
     "docs-compile": "npm run docs-build",
-    "docs-linkinator": "linkinator _site --recurse --skip \"^(?!http://localhost)\" --verbosity error",
+    "docs-linkinator": "npx linkinator _site --recurse --skip \"^(?!http://localhost)\" --verbosity error",
```

Comment:
I'm thinking perhaps we should remove the npm script completely...

Reply:
I don't think so, we also have spellcheck and validator here so it makes sense IMHO. Or we may drop all of them ?

Reply:
My plan is to move those to Actions too, but they are not exactly the same in my eyes. cpell can run locally out of the box, linkinator requires a docs build first.

I'm in the middle TBH. I just want to reduce our devDependencies since we do have CI.

Reply:
Also, do note that using `npx` is not considered a good practice for good reasons.

Reply:
Then I guess vnu could be dropped too in our devDeps, if it runs in an action already? It also requires a build step I think, and is probably one of our heaviest dependency.

Whatever, drop the linkinator script. This is indeed covered by CI. 👌

Reply:
Unfortunately, there's no ready Action to run vnu AFAICT. If there was, I would consider moving that to Actions too.

Reply:
So, I thought a bit about this and perhaps the best solution would be to remove the `linkinator` script from the `docs-lint` script. It will still be there for now, just not used.

Reply:
Will we really use it, knowing it's covered by CI? I'm not against keeping it but it looks like dead code if it's not used, isn't it?

Reply:
I was thinking to keep it until we are sure we are OK with the current approach, but maybe it's fine to just drop it...